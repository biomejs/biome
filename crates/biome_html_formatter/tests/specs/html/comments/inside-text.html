foo bar baz floof
<!-- comment -->
quick brown fox
