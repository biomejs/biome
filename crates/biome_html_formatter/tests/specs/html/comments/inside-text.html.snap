---
source: crates/biome_formatter_test/src/snapshot_builder.rs
info: comments/inside-text.html
---
# Input

```html
foo bar baz floof
<!-- comment -->
quick brown fox

```


=============================

# Outputs

## Output 1

-----
Indent style: Tab
Indent width: 2
Line ending: LF
Line width: 80
Attribute Position: Auto
Bracket same line: false
Whitespace sensitivity: css
Indent script and style: false
Self close void elements: never
-----

```html
foo bar baz floof
<!-- comment -->
quick brown fox
```
